filer/private/sass/components/_drag-and-drop.scss

```diff
@@ -358,8 +366,8 @@ form .form-row {
         margin-right: 10px;
         border: solid 1px $gray-lighter;
         border-radius: 3px;
-        background: $white url("../icons/file-unknown.svg");
-        background-size: cover;
+        background: $white url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8' standalone='no'%3F%3E%3Csvg xmlns:dc='http://purl.org/dc/elements/1.1/' xmlns:cc='http://creativecommons.org/ns%23' xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns%23' xmlns:svg='http://www.w3.org/2000/svg' xmlns='http://www.w3.org/2000/svg' xmlns:sodipodi='http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd' xmlns:inkscape='http://www.inkscape.org/namespaces/inkscape' inkscape:version='1.0beta2 (2b71d25, 2019-12-03)' sodipodi:docname='file.svg' id='svg22' version='1.1' width='24' height='24'%3E%3Cmetadata id='metadata28'%3E%3Crdf:RDF%3E%3Ccc:Work rdf:about=''%3E%3Cdc:format%3Eimage/svg+xml%3C/dc:format%3E%3Cdc:type rdf:resource='http://purl.org/dc/dcmitype/StillImage' /%3E%3Cdc:title%3E%3C/dc:title%3E%3C/cc:Work%3E%3C/rdf:RDF%3E%3C/metadata%3E%3Cdefs id='defs26' /%3E%3Csodipodi:namedview inkscape:current-layer='svg22' inkscape:window-maximized='0' inkscape:window-y='28' inkscape:window-x='0' inkscape:cy='7.820029' inkscape:cx='12' inkscape:zoom='28.708333' showgrid='false' id='namedview24' inkscape:window-height='776' inkscape:window-width='1440' inkscape:pageshadow='2' inkscape:pageopacity='0' guidetolerance='10' gridtolerance='10' objecttolerance='10' borderopacity='1' inkscape:document-rotation='0' bordercolor='%23666666' pagecolor='%23ffffff' /%3E%3Cg id='g56'%3E%3Cpath inkscape:connector-curvature='0' id='path10' fill='%232980b9' d='M 5,2 C 3.8954,2 3,2.9 3,4 v 8 4 6 c 0,1.1 0.8954,2 2,2 h 14 c 1.105,0 2,-0.9 2,-2 V 16 12 8 L 15,2 Z' /%3E%3Cpath inkscape:connector-curvature='0' id='path12' fill='%233498db' d='M 5,1 C 3.8954,1 3,1.9 3,3 v 8 4 6 c 0,1.1 0.8954,2 2,2 h 14 c 1.105,0 2,-0.9 2,-2 V 15 11 7 L 15,1 Z' /%3E%3Cpath inkscape:connector-curvature='0' id='path14' fill='%232980b9' d='M 21,7 15,1 v 4 c 0,1.1 0.895,2 2,2 z' /%3E%3C/g%3E%3C/svg%3E%0A");
```

Comment:
We might replace this "heavy" svg code by this lighter one:

```html
data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24'%3E%3Cpath fill='%232980b9' d='M5 2c-1.105 0-2 .9-2 2v18c0 1.1.895 2 2 2h14c1.105 0 2-.9 2-2V8l-6-6z'/%3E%3Cpath fill='%233498db' d='M5 1c-1.105 0-2 .9-2 2v18c0 1.1.895 2 2 2h14c1.105 0 2-.9 2-2V7l-6-6z'/%3E%3Cpath fill='%232980b9' d='m21 7-6-6v4c0 1.1.895 2 2 2z'/%3E%3C/svg%3E
```

(I used svgoptimizer.com to reduce the file size & [this project](https://yoksel.github.io/url-encoder/) to encode it for browsers)